### PR TITLE
VSRevoGroup.RevoUninstallerPro version 5.0.1.0

### PR DESCRIPTION
**Pull request by GuyInCanada**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60119)

### CODE REVIEW

**Comment by tim-mann** [comment]
The installer URL is from here

https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This should be moved to `RevoUninstaller.RevoUninstallerPro` and use existing metadata from `RevoUninstaller.RevoUninstallerPro` version `4.4.8`

`PackageVersion` also needs to be `5.0.1` instead of `5.0.1.0` to match Control Panel version.

File: manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+PackageIdentifier: RevoUninstaller.RevoUninstallerPro
+PackageVersion: 5.0.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://1baf5346ec1e2437e714-1959a2065c6dc5b4e5e9f6fa5eeb4840.ssl.cf1.rackcdn.com/RevoUninProSetup.exe
+  InstallerSha256: 9F7AD647D126310E850D64F871776462FE359B530CE12CF9155E4E94DE71133A
+ManifestType: installer
+ManifestVersion: 1.1.0
+
```
Comment (ItzLevvie) [inline manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.installer.yaml:13]:
```suggestion
PackageIdentifier: RevoUninstaller.RevoUninstallerPro
PackageVersion: 5.0.1
MinimumOSVersion: 10.0.0.0
InstallModes:
- interactive
- silent
- silentWithProgress
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://1baf5346ec1e2437e714-1959a2065c6dc5b4e5e9f6fa5eeb4840.ssl.cf1.rackcdn.com/RevoUninProSetup.exe
  InstallerSha256: 9F7AD647D126310E850D64F871776462FE359B530CE12CF9155E4E94DE71133A
  UpgradeBehavior: install
ManifestType: installer
ManifestVersion: 1.1.0

```

File: manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: RevoUninstaller.RevoUninstallerPro
+PackageVersion: 5.0.1
+PackageLocale: en-US
+Publisher: VS Revo Group
+PackageName: Revo Uninstaller Pro
+License: Proprietary
+Copyright: VS Revo Group, Ltd.
+ShortDescription: Revo Uninstaller Pro
+ManifestType: defaultLocale
+ManifestVersion: 1.1.0
```
Comment (ItzLevvie) [inline manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.locale.en-US.yaml:13]:
```suggestion
PackageIdentifier: RevoUninstaller.RevoUninstallerPro
PackageVersion: 5.0.1
PackageLocale: en-US
Publisher: VS Revo Group, Ltd.
PublisherUrl: https://www.revouninstaller.com
PublisherSupportUrl: https://www.revouninstaller.com/support
PrivacyUrl: https://www.revouninstaller.com/revo-uninstaller-privacy-policy
Author: VS Revo Group, Ltd.
PackageName: Revo Uninstaller Pro
PackageUrl: https://www.revouninstaller.com/products/revo-uninstaller-pro
License: Proprietary
LicenseUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
Copyright: Copyright VS Revo Group
CopyrightUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
ShortDescription: Revo Uninstaller helps you to uninstall software and remove unwanted programs easily.
Description: Revo Uninstaller is a freeware uninstall utility. It has powerful features to uninstall programs scanning for leftover files, folders and registry entries after uninstall. With its unique 'Hunter mode' it offers you some simple, easy to use but effective and powerful approaches to manage (uninstall, stop, delete, disable from auto starting) and to get information about your installed and/or running application. Many cleaning tools included!
Moniker: revo-pro
Tags:
- revo
- revo-uninstaller-pro
- cleaner
ManifestType: defaultLocale
ManifestVersion: 1.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
File names and folder names will need to be renamed to match PackageIdentifier and PackageVersion.

**Pull request closed without merging**